client/jetpack-connect/plans.jsx

```diff
@@ -124,11 +110,13 @@ class Plans extends Component {
 		} else if ( props.selectedSite ) {
 			this.props.goBackToWpAdmin( props.selectedSite.URL + JETPACK_ADMIN_PATH );
 		}
+		this.redirecting = true;
 		this.props.completeFlow();
```

Comment:
The conditions aren't total, we could arrive here without entering either conditional branch. The `redirecting` and `completeFlow` should likely be in each branch.

Reply:
Well spotted 👍